src/main/resources/application-production.properties

```diff
@@ -2,7 +2,8 @@ spring.datasource.url=${JDBC_DATABASE_URL}
 spring.datasource.username=${JDBC_DATABASE_USERNAME}
 spring.datasource.password=${JDBC_DATABASE_PASSWORD}
 
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+spring.h2.console.enabled=${H2_CONSOLE_ENABLED:${env.H2_CONSOLE_ENABLED:false}}
```

Comment:
This doesn't belong in this PR.  Please revert this file to what's on the main branch.   These other changes should get made in the appropriate PR, *not in this one*.

Reply:
I've removed the file, it came out of my attempt to rebase (I wasn't entirely familiar with how rebasing worked). The file shouldn't be changed on this PR anymore